libc-bottom-half/sources/__main_void.c

```diff
@@ -3,6 +3,7 @@
 #include <sysexits.h>
 
 // The user's `main` function, expecting arguments.
+__attribute__((__weak__, nodebug))
```

Comment:
why nodebug?

Reply:
I probably copied the whole line from elsewhere; I'll remove that part.